src/sdk/http/http-client.ts

```diff
@@ -156,7 +158,29 @@ export abstract class HttpClient<TRequest, TResponse> implements IHttpClient {
 			}
 		});
 	}
-
+	
+	getResponse(path: string, params?: any): Promise<any> {
```

Comment:
Why still returning the raw `response` rather than the abstract method I proposed earlier? Was there a reason that wouldn't have worked? This still requires some internal implementation knowledge for consumers.